doc/release-notes/release-7.sgml.in

```diff
@@ -30,11 +30,13 @@ The Squid-@SQUID_RELEASE@ change history can be <url url="https://github.com/squ
 
 <p>The most important of these new features are:
 <itemize>
-	<item>
+	<item>removed support for the cache_object URI scheme
 </itemize>
 
 Most user-facing changes are reflected in squid.conf (see below).
 
+<sect1>cache_object scheme removal
+<p>Squid cache manager is no longer accessible by URIs with cache_object scheme.
```

Comment:
I tried to polish this a little. If anybody does not like the current wording, please just edit the file to your liking. I do not insist on any specific wording here.